org-formation/080-aws-config-inventory/_tasks.yml

```diff
@@ -0,0 +1,24 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'config'
+
+# Enable AWS Config in all member accounts and send Findings and Config history to the centralized s3 bucket in the archive account
+ConfigBase:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/Config/config.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-base'
+  StackDescription: AWS Config - Base
+  TerminationProtection: false
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+  OrganizationBindings:
+    LogArchiveBinding:
+      Account: !Ref LogCentralAccount
+  Parameters:
+    resourcePrefix: !Ref resourcePrefix
+    bucketName: !Sub '${resourcePrefix}-${appName}-${CurrentAccount.AccountId}'
```

Comment:
What will be the the actual bucket names? Looks like logs for each account will be written to different buckets within logcentral. Is that how we want it for Sumo?

Reply:
config logs works like cloudtrail.  there will be a bucket in each account with the above name, the distinction is the AccountId.  The bucket in logcentral will contain an aggregate of all accounts and that's the one that we will setup with Sumo.